process_request/helpers.py

```diff
@@ -375,10 +375,11 @@ def get_parent_title(obj_json):
     return title
 
 
-def get_url(obj_json, host, client):
+def get_url(obj_json, client, host=None):
```

Comment:
The doc says this returns a "full URL", which is not always true.  It could return a full URL or a relative URI, depending on if the host is passed.